migrations/migrations/20210105232005-users.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
       password: {
         type: Sequelize.STRING,
         allowNull: false
+      },
+      role: {
+        type: Sequelize.ENUM,
+        values: ['admin', 'user']
```

Comment:
You should not modify the previous migration but create a new one that adds the new field